lib/modules/platform/gitea/index.spec.ts

```diff
@@ -1722,19 +1720,17 @@ describe('modules/platform/gitea/index', () => {
         number: 42,
         title: 'pr-title',
       });
-      expect(mergePR).toHaveBeenCalled();
+      expect(mergePR).not.toHaveBeenCalled();
     });
 
-    it('should use platform automerge on forgejo v7 LTS', async () => {
+    it('should not use platform automerge on forgejo v7 LTS', async () => {
       memCache.set('gitea-pr-cache-synced', true);
       const helper = await import('./gitea-helper');
       const mergePR = jest.spyOn(helper, 'mergePR');
       const scope = httpMock
         .scope('https://gitea.com/api/v1')
         .post('/repos/some/repo/pulls')
-        .reply(200, mockNewPR)
-        .post('/repos/some/repo/pulls/42/merge')
-        .reply(200);
+        .reply(200, mockNewPR);
       await initFakePlatform(scope, '7.0.0+LTS-gitea-1.22.0');
```

Comment:
`7.0.0+gitea-1.22.0` is how `https://code.forgejo.org/api/v1/version` would return the version.

Reply:
there was a time i saw it coming from the api and it doesn't matter for new checks. we now rely on  there semver before the meta `+`. the meta is only used to detect forgejo. will soon split the platforms for easier maintenance as tehy will diverge much this year.